grpc/tls/provider/meshca/v1/service.proto

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grpc.tls.provider.meshca.v1;
```

Comment:
This looks like it should be in the googleapis repository. Since it is GCP-specific, that seems the more appropriate place. There's other protos in googleapis that aren't intended to be used directly and have fat clients (e.g., bigtable).

Reply:
This is something that we can discuss tomorrow.

Reply:
This is removed now.